src/System.Numerics.Tensors/src/Resources/Strings.resx

```diff
@@ -0,0 +1,168 @@
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ArrayMustContainElements" xml:space="preserve">
+    <value>Array must contain elements.</value>
+  </data>
+  <data name="CannotCompare" xml:space="preserve">
+    <value>Cannot compare {0} to {1}.</value>
+  </data>
+  <data name="CannotCompareToWithDifferningDimension" xml:space="preserve">
+    <value>Cannot compare {0} to {1} with differning dimension {2}, {3} != {4}.</value>
+  </data>
+  <data name="CannotCompareWithDifferningDimension" xml:space="preserve">
+    <value>Cannot compare {0}s with differning dimension {1}, {2} != {3}.</value>
```

Comment:
`differning ` should be `differing` or better, `different`

Also we cannot pluralize in string formatting. It won't localize because when they translate the string, they do not know what the noun is to get the pluralization right. We (assume we) get away with it when we quote things, or avoid dynamic plurals. Eg.,

`<value>Cannot compare two '{0}' with different {1} dimensions. Lengths are {2} and {3}.</value>`

Several strings here have this problem.

Reply:
Should I change `Cannot compare {0} with different dimension {1}, {2} != {3}.` to `Cannot compare two '{0}' with different {1} dimensions. Lengths are {2} and {3}.`?